Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -632,6 +638,10 @@ WI.initializeTarget = function(target)
                 // FIXME <rdar://105244623> iOS platforms without a matching legacy protocol definition will fall back to the local macOS protocol definition.
                 console.error(error);
             });
+
+        // COMPATIBILITY (iOS 18.0): Page.Setting.SiteIsolationDebugOverlay did not yet exist.
+        if (target.hasCommand("Page.overrideSetting") && InspectorBackend.Enum.Page.Setting.SiteIsolationDebugOverlay)
```

Comment:
yeah i just meant `InspectorBackend.Enum.Page.Setting.SiteIsolationDebugOverlay`

as you pointed out, i think `Page.overrideSetting` exists for all supported versions, so im not sure that part of this check is necessary